cmd/vic-machine/create/create.go

```diff
@@ -692,6 +695,15 @@ func (c *Create) Run(clic *cli.Context) (err error) {
 		return err
 	}
 
+	if c.StorageQuotaGB != nil {
+		err = validator.ValidateStorageQuota(op, *c.StorageQuotaGB, nil, false)
```

Comment:
I put it into Validate initially, and found both create and configure will invoke validator.Validate() func. But there are different behaviors between create and configure, so I move them out of Validate.

Reply:
Where ever possible we've kept symmetry between create and configure to avoid needing special case logic, duplicated code to support migration of old configuration versions, etc.

Can we check for the existence of the expected datastore paths in the `ValidateStorageQuota` method and only check for violation if the path exists? This will become more significant when adding volume store quota support, givenvolume stores can be added during configure, but will not exist until the endpointVM boots and initializes them.

Reply:
We may refactor, expand, or create new validators when it comes to volume quota validations. For now, I would suggest to keep it just for storage quota.

Reply:
It's important that all validation happens as a part of the `Validate` method so that the list of errors returned is complete.

Reply:
I revisited that part of codes, and think we may need to refactor Validate api in a separate patch if we still want to put ValidateStorageQuota into it, as there are two new parameters for this api, and if only add them into Validate api's parameters, the api will get many input parameters eventually.